lib/plugins/aws/utils/iam.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.getDeploymentRole = provider => {
+  if (provider.iam && provider.iam.deploymentRole) {
+    return provider.iam.deploymentRole;
+  }
+
+  if (provider.cfnRole) {
+    // Warn?
```

Comment:
Here we should log deprecations as outlined in: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md#configuring-deprecations